br/pkg/lightning/errormanager/errormanager_test.go

```diff
@@ -248,6 +248,42 @@ func TestReplaceConflictKeysIndexKvChecking(t *testing.T) {
 	tbl, err := tables.TableFromMeta(tidbkv.NewPanickingAllocators(0), table)
 	require.NoError(t, err)
 
+	sessionOpts := encode.SessionOptions{
+		SQLMode:   mysql.ModeStrictAllTables,
+		Timestamp: 1234567890,
+	}
+
+	encoder, err := tidbkv.NewBaseKVEncoder(&encode.EncodingConfig{
+		Table:          tbl,
+		SessionOptions: sessionOpts,
+		Logger:         log.L(),
+	})
+	require.NoError(t, err)
+	encoder.SessionCtx.GetSessionVars().RowEncoder.Enable = true
+
+	data1 := []types.Datum{
+		types.NewIntDatum(1),
+		types.NewIntDatum(6),
+		types.NewStringDatum("1.csv"),
+	}
+	data2 := []types.Datum{
+		types.NewIntDatum(2),
+		types.NewIntDatum(6),
+		types.NewStringDatum("2.csv"),
+	}
+	_, err = encoder.Table.AddRecord(encoder.SessionCtx, data1)
+	require.NoError(t, err)
+	_, err = encoder.Table.AddRecord(encoder.SessionCtx, data2)
+	require.NoError(t, err)
+	kvPairs := encoder.SessionCtx.TakeKvPairs()
+
+	rawKey := kvPairs.Pairs[1].Key
```

Comment:
I'd like to rename these variables like `data1RowKey`, `data2IndexValue` so reviewers can easier to remember their meaning.